lblod/frontend-lokaal-mandatenbeheer
# Lmb 796 export mandatarissen as csv

## Description

We want the mandatarissen to be downloadable. The file includes the required columns and the data will be filtered the way it was in the frontend. Creating this export is done in the mandataris service

## How to test

Open the mandataris search page or the organen>mandatarissen page and download the content. See if the results have the active filters applied. You will will also see previous mandatarissen of the person but this is how we want it.

## Links to other PR's

- https://github.com/lblod/mandataris-service/pull/55

## Attachments

![image](https://github.com/user-attachments/assets/1c690f53-f5cb-41d3-a881-e0f1462e3bd9)
